gamification/process/update.py

```diff
@@ -135,6 +137,42 @@ def get_participant_objects():
     return participants
 
 
+def get_meta_review_participant_objects():
+    return meta_review_participants.objects.all()
+
+
+meta_review_completed_list = []
+
+
+def update_participants_data_with_meta_review(meta_review_participant):
+    """
+    Update participants based on meta-review
+
+    This method updates every participant based on the meta-review
+    received or given. If the gamification participant is
+    in the active newcomers list then it will check if the meta-review
+    is complete or not and update the activity accordingly.
+    """
+    active_newcomers_list = active_newcomers()
+    if (meta_review_participant.login in active_newcomers_list and
+            meta_review_participant.login not in meta_review_completed_list):
+        # Get the corresponding gamification participant
+        gamification_participant = Participant.objects.get(
+            username=meta_review_participant.login)
+        reaction_name = Reaction.objects.get(id=meta_review_participant.login)
+        if (meta_review_participant.pos_in > 0 or
+                meta_review_participant.pos_out > 0 or
+                meta_review_participant.neg_in > 0 or
+                meta_review_participant.neg_out > 0):
+            # Add a meta-review activity to the participant
+            activity = 'Did a meta-review or received a meta-review'
+            created_at = reaction_name.created_at
+            updated_at = None
+            gamification_participant.add_activity(0, activity, created_at,
+                                                  updated_at)
```

Comment:
Maybe add an `info level` log here saying that `this participant(get the username of the participant with gamification_participant.username ) has performed meta-review activity`.